build-system/tasks/compile.js

```diff
@@ -295,6 +295,8 @@ function compile(entryModuleFilenames, outputDir,
           '!build/fake-module/src/polyfills/**/*.js',
           '!build/fake-polyfills/src/polyfills.js',
           '!src/polyfills/*.js',
+          // TODO(prateekbh): I don't understand how to add
+          // src/polyfills/custom-elements.js to the _needed_ polyfills.
```

Comment:
@prateekbh

Reply:
do we need to exclude `src/polyfills/custom-elements.js` for the module build?

cuz last time we checked with Malte, the lifecycle without custom elements polyfills is skewed.

Reply:
No, we need to include it in all builds. I don’t know how to do that for the module build.

Reply:
By default it will be added in all.
Extra work is for exclusion

Reply:
Are you sure? The previous line (`'!src/polyfills/*.js'`) excludes everything from polyfills directory.

Reply:
Oh! with your PR we wont need to shadow it anywhere else, gotcha.

best way would be to remove the above line and add a code below this
```
polyfillsShadowList.forEach(polyfillFile => {
  srcs.push(`!src/polyfills/${polyfillFile}.js`);
})
```

this will exclude all but your polyfill